doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/user-guides/autoscaling.md

```diff
@@ -0,0 +1,91 @@
+(kuberay-autoscaler-discussion)=
+# Autoscaling
```

Comment:
The sectioning structure in the second half of this doc is pretty confusing, should it be fixed and/or flattened a bit?

Reply:
hashtag typos, fixing...

Reply:
more hashtags added